src/HashBus.Twitter.BackFill/App.cs

```diff
@@ -1,26 +1,30 @@
 ﻿namespace HashBus.Twitter.BackFill
 {
+    using System.Threading;
+    using System.Threading.Tasks;
     using HashBus.NServiceBusConfiguration;
     using HashBus.Twitter.Monitor.CatchUp.Commands;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static void Run(
-            string nserviceBusConnectionString,
-            string endpointName,
-            string track,
-            long tweetId)
+        public static async Task Run(string nserviceBusConnectionString, string endpointName, string track, long tweetId, string monitorCatchUpAddress)

```

Comment:
Please reformat the parameters to have one parameter on each line as previously.